stubs/olefile/olefile/__init__.pyi

```diff
@@ -0,0 +1,122 @@
+import datetime
```

Comment:
At runtime this file just has `from .olefile import *`. The stub should do the same. Currently, it instead duplicates all the definitions from `olefile.olefile`.

Reply:
Thanks, I've pushed a commit for this.